sklearn/mixture/dpgmm.py

```diff
@@ -106,6 +116,8 @@ def _bound_state_log_lik(X, initial_bound, precs, means, covariance_type):
     return bound
 
 
+@deprecated("The class DPGMM is deprecated in 0.18 and "
+            "will be removed in 0.20. This class no longer works.")
```

Comment:
Can you change the message in that way "The DPGMM class is not working correctly and it's better to not use it. DPGMM is deprecated in..."